src/main/java/org/humancellatlas/ingest/project/ProjectRepository.java

```diff
@@ -41,8 +41,6 @@ Page<Project> findByUserOrPrimaryWrangler(@Param(value = "user") String user,
                                                 @Param(value = "primaryWrangler") String primaryWrangler, 
                                                 Pageable pageable);
 
-    Page<Project> findBySubmissionEnvelope(SubmissionEnvelope submissionEnvelope, Pageable pageable);
```

Comment:
I forgot the reason for deleting these methods from the repository. Can you please remind me?

Reply:
Oh, I read it in the description now :)

Reply:
This repository method gives you the "child" projects contained within a submission envelope, so should not be required.

The one case where it could possibly be required is when you upload a spreadsheet with project tabs to an existing project. I can check the broker code and test this on the dev ui before we merge to master.

Reply:
Having looked through the broker/client code,
On import any spreadsheet that contains a [project with a project uuid](https://github.com/ebi-ait/ingest-client/blob/bd5d2353849ff8e72d15318d3cf21f41683d7786/hca_ingest/importer/importer.py#L280) is considered [an *existing entity*](https://github.com/ebi-ait/ingest-client/blob/bd5d2353849ff8e72d15318d3cf21f41683d7786/hca_ingest/importer/importer.py#L198), since only [new entities](https://github.com/ebi-ait/ingest-client/blob/bd5d2353849ff8e72d15318d3cf21f41683d7786/hca_ingest/importer/submission/entity.py#L48) are [inserted into ingest](https://github.com/ebi-ait/ingest-client/blob/bd5d2353849ff8e72d15318d3cf21f41683d7786/hca_ingest/importer/importer.py#L82) with the submission envelope, but an [update to the existing project is made](broker/importer#85)